src/zac/conf/settings.py

```diff
@@ -13,10 +13,16 @@
 import os
 
 from django.urls import reverse_lazy
+from .local_example import *
+
+
+TEST_PLATFORM_URL = 'https://vng-staging.maykin.nl'
```

Comment:
I know that you're working on it, but our demo has no notion or dependency on the test platform. This setting does not belong here.